src/lib/proposalUtils.ts

```diff
@@ -23,7 +24,11 @@ export function importUrlValues<Values>(defaultValues: Values) {
           initialFormValues[prop] = BigInt(paramValue);
           break;
         case "object":
-          initialFormValues[prop] = JSON.parse(paramValue);
+          if (moment.isMoment(defaultValues[prop])) {
+            initialFormValues[prop] = moment(paramValue);
+          } else {
+            initialFormValues[prop] = JSON.parse(paramValue);
```

Comment:
how this is related to this pr ?

Reply:
Competition parameters include dates.  This is to support them.